test/integration/serverTest.js

```diff
@@ -864,8 +1024,8 @@ describe('Brave Sensor server', () => {
 
     describe('POST /locations/:locationId', () => {
       beforeEach(async () => {
-        sinon.stub(helpers, 'log')
-        sinon.stub(helpers, 'logError')
+        sinon.spy(helpers, 'log')
+        sinon.spy(helpers, 'logError')
```

Comment:
I found it useful when debugging these tests to have the logs still be printed out to the console, so I changed this from stub to spy.